Gemfile.lock

```diff
@@ -61,19 +63,18 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.3.3)
-      actionpack (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
+    railties (6.1.3)
+      actionpack (= 6.1.3)
+      activesupport (= 6.1.3)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.20.3, < 2.0)
-    rake (13.0.1)
-    thor (1.0.1)
-    thread_safe (0.3.6)
-    tzinfo (1.2.7)
-      thread_safe (~> 0.1)
-    zeitwerk (2.4.0)
-    zxcvbn-ruby (1.1.0)
+      thor (~> 1.0)
+    rake (13.0.3)
+    thor (1.1.0)
+    tzinfo (2.0.4)
+      concurrent-ruby (~> 1.0)
+    zeitwerk (2.4.2)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.4.0 → 2.4.2

[_[change-log](https://github.com/fxn/zeitwerk/blob/master/CHANGELOG.md), [source-code](https://github.com/fxn/zeitwerk)_]





<details>
<summary>Commits</summary>

A change of **12** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.4.0...v2.4.2).

These are the first 10 commits:
- (90620a1) [fixes typo](https://github.com/fxn/zeitwerk/commit/90620a189d41200720e7a0c9f1e0cd8f7b793df8)
- (8167a8d) [Use __dir__ in collapse examples [Closes #131]](https://github.com/fxn/zeitwerk/commit/8167a8dcd6b6dcc5c625d701836282f9c19ef7ca)
- (f892e80) [Add truffleruby-head to CI](https://github.com/fxn/zeitwerk/commit/f892e800e5dfdc879bf046922c89703bb5da7dae)
- (1790e82) [Merge pull request #132 from bjfish/truffleruby-ci](https://github.com/fxn/zeitwerk/commit/1790e82c579e118522c181e37c3f4870e17deb56)
- (627482a) [removes truffleruby-head from allowed CI failures 🎉](https://github.com/fxn/zeitwerk/commit/627482af33c895c7199b261e78f4d8037b01938e)
- (60021d9) [RBS notation for methods and attributes](https://github.com/fxn/zeitwerk/commit/60021d90af973d98ee19e9634a552b25c39320cc)
- (cce3c39) [Use __send__ instead of send](https://github.com/fxn/zeitwerk/commit/cce3c392e0de92569529bffac46e26cc0e716805)
- (26f232e) [Merge pull request #140 from mfilej/send](https://github.com/fxn/zeitwerk/commit/26f232e21c8faad5d5d1143dcc732845ff6db569)
- (034ae30) [version 2.4.1](https://github.com/fxn/zeitwerk/commit/034ae30d73247b8dda7df2992903ce560cd7f47f)
- (c716367) [Implements Zeitwerk::Loader#on_load](https://github.com/fxn/zeitwerk/commit/c71636797544a649eb6cb40bcc516736bb57d3d4)


</details>